### ISSUE TITLE
[Question] Recipe for when whole app is not in React

### ISSUE THREAD

**Issue by binarykitchen** [gaearon/react-hot-loader#694]
When researching for a while, I couldn't find a solution for my following problem and hope one of you can show me the "light"

I have a legacy app here based on old technologies. Only a few directories of that app are written in React. These are then wrapped in containers to make it work with the older app. Now with that, all my attempts to include the react-hot-loader have failed. Always seeing that infamous "Ignored an update to unaccepted module" warning.

Let me rephrase the problem in other words: The root container is based on Ext.BoxComponent (ExtJS) which includes a React component called "Archiving", created by the `React.createFactory()` fn. How can I tell the hot loader to reload when the Archiving component of one of its children has changed?

**Comment by theKashey** [comment]
Which version or react-hot-loader are you using?

**Comment by binarykitchen** [comment]
The latest one, 3.1.1

**Comment by theKashey** [comment]
So, and how do it use it - with webpack loader, or with a babel plugin?
Could you provide a real warning, and a code example for it.

**Comment by binarykitchen** [comment]
Think I figured it out myself. It's actually a webpack HMR issue itself. Sorry.
